content-src/selectors/siteMetadataSelectors.js

```diff
@@ -8,7 +8,12 @@ function selectSiteProperties(site) {
   const metadataFavicon = site.favicons && site.favicons[0] && site.favicons[0].url;
   const favicon = site.favicon_url || metadataFavicon || site.favicon;
   const parsedUrl = site.parsedUrl || urlParse(site.url || "");
-  const label = prettyUrl(parsedUrl.hostname);
+
+  // Remove the eTLD (e.g., com, net) and the preceding period from the hostname
+  const eTLDLength = (site.eTLD || "").length;
```

Comment:
Do you need the `||` if [here](https://github.com/mozilla/activity-stream/pull/1377/files#diff-49c53f46436babc1b5d680484d645cdcR477) you are assigning in both cases?